.mvn/extensions.xml

```diff
@@ -2,6 +2,6 @@
   <extension>
     <groupId>io.jenkins.tools.incrementals</groupId>
     <artifactId>git-changelist-maven-extension</artifactId>
-    <version>1.2</version>
+    <version>1.4</version>
```

Comment:
Unrelated code cleanup done while I was here: bumping the `git-changelist-maven-extension` to the latest version to facilitate producing incremental builds for testing purposes.